flutter/Graminsta/lib/createPostWidget.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart';
```

Comment:
Usually, to reduce the burden for reviewers, try to split code into at least backend and frontend. But do that in following PRs, not this one.